scenario.py

```diff
@@ -71,10 +72,29 @@ def __init__(self, params, experiment_path):
 
         self.contributivity_list = []
 
-        if 'epoch_count' in params.keys():
+        if 'epoch_count' in params:
             self.epoch_count = params['epoch_count']
         else:
             self.epoch_count = 40
+            
+        methods_default = [
+            "Shapley values",
+            "Independant scores",
+            "TMCS values"
+        ]
+        
+        self.methods = []
+        if not 'methods' in params:
+            self.methods = methods_default
+        else:
+            if not params['methods']:
+                sys.exit('No contributivity method given in config file')
+            else:
+                for el in params['methods']:
+                    if el in methods_default:
+                        self.methods.append(el)
+                    else:
+                        sys.exit('method ' + el + ' is not in methods list.')
```

Comment:
I would raise an exception here instead of exit since this is not a "normal" exit:
`raise Exception("No contributivity method given in config file")`

Reply:
Yes, that's probably better! I changed it.